src/__snapshots__/App.test.js.snap

```diff
@@ -20,6 +20,12 @@ exports[`page renders correctly 1`] = `
     </p>
     <p>
       Ethan Moliver | CMD | Spicy char kway teow | Community
+<<<<<<< HEAD
```

Comment:
Actually looks like you left one of the merge conflicts in

Reply:
yeah made an error there

Reply:
should be all good now